src/ClusterBootstrap/params.py

```diff
@@ -751,28 +751,7 @@
     "custom_mounts": [],
     "enable_blobfuse": False,
 
-    # To use CPU nodes,
-    # 1. CPU nodes must have node label cpuworker=active
-    # 2. enable_cpuworker is set to True
-    # 3. default_cpu_sku is set to a valid value that exists in sku_meta
-    "enable_cpuworker": False,
     "enable_custom_registry_secrets": False,
-    "default_cpu_sku": "Standard_D2s_v3",
-
-    # SKU meta defines different types of resources for each SKU
-    # and their allowed usage ratio by user applications.
-    "sku_meta": {
```

Comment:
@YinYangOfDao can you help confirm this is no longer used? I see the new config is `sku_mapping`.

Reply:
yes, confirmed